modules/geo-layers/src/mvt-layer/mvt-layer.ts

```diff
@@ -22,28 +38,80 @@ const defaultProps = {
   binary: true
 };
 
-export default class MVTLayer extends TileLayer {
-  initializeState() {
+export type TileJson = {
+  tilejson: string;
+  tiles: string[];
+  // eslint-disable-next-line camelcase
+  vector_layers: any[];
+  attribution?: string;
+  scheme?: string;
+  maxzoom?: number;
+  minzoom?: number;
+  version?: string;
+};
+
+/** All props supported by the MVTLayer */
+export type MVTLayerProps<DataT extends Feature = Feature> = _MVTLayerProps &
+  GeoJsonLayerProps<DataT> &
+  TileLayerProps<DataT>;
```

Comment:
Isn't `TileLayerProps`'s `DataT` the type of tile data?

Reply:
I am really not sure.
Looks i've misunderstood `TileLayer`s `DataT` when converting originally. Will remove this and think about cleaning up this typings.

Anyway, going this direction, it should be `TileLayerProps<Geometry[] | GeometryCollection | BinaryAttributes>` ? I'm voting for next PR.

Reply:
I went with `Feature[] | BinaryAttributes` as it looks like it's ok.